packages/mask/src/resources/PangolinIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import { SvgIconProps, SvgIcon } from '@mui/material'
+
+const svg = (
+    <svg xmlns="http://www.w3.org/2000/svg" fill="#FF6B00" viewBox="0 0 384 387">
```

Comment:
plz use https://aka.pw/svgomg reduce this svg image

Reply:
2.73k → 1.35k